src/Modules/OrchardCore.Commerce/Handlers/TaxPartAndPricePartHandler.cs

```diff
@@ -39,7 +39,7 @@ public override async Task UpdatedAsync(UpdateContentContext context, PricePart
         var taxRate = taxPart.TaxRate?.Value ?? 0;
 
         var isGrossPricePresent = taxPart.GrossPrice?.Amount.IsValid == true;
-        var isTaxRatePresent = taxRate > 0;
+        var isTaxRatePresent = taxRate >= 0;
```

Comment:
This is a bad change, because now `isTaxRatePresent` is always true, unless the admin manually enters a negative number. I've provided the fixed logic in #704.